<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #495057;</string>
  </property>
  <widget class="QPushButton" name="BTNDownloder">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #edf2f4;
font: 75 italic 15pt &quot;Book Antiqua&quot;;
color:rgb(113, 113, 113)</string>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="URLField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 162, 162);
font: italic 12pt &quot;Monotype Corsiva&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter Youtube URl</string>
   </property>
  </widget>
  <widget class="QLabel" name="MessageFiled">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;

background-color: #8d99ae;
color:#ffffff;</string>
   </property>
   <property name="text">
    <string>Lets Download Youtube Video</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="ListDirFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>431</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8d99ae;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Dir_List">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9999a1;
font: italic 12pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="FilesList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 12pt &quot;Monotype Corsiva&quot;;
background-color: rgb(235, 235, 235);</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="YoutubeTitleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffffff;;
text-decoration: underline;
font: 25 18pt &quot;Bahnschrift Light&quot;;
</string>
   </property>
   <property name="text">
    <string>Youtube Video Downloder</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
